stay-sweeper.js

```diff
@@ -1,22 +1,21 @@
 const mongoose = require ('mongoose');
+const sitnpaws = require('./db/config');
 mongoose.Promise = global.Promise;
 const Stay = require('./db/models/stays');
 
 let today = new Date();
 
-const sweepStays = (() => {
-  Stay.find({}).where('endDate').lt(today).exec().then((stays) => {
-    stays.forEach((stay) => {
-      if (stay.status === 'pending') {
-        stay.update({'status': 'expired'}).exec();
-      } else if (stay.status === 'approved') {
-        stay.update({'status': 'complete'}).exec();
-      }
-    });
+const sweepStays = () => {
+  const expiringStays = Stay.find({}).where('startDate').gt(today).where('status').equals('pending').exec();
```

Comment:
Looks like it should be: `where('startDate').gt(today)` to apply to start dates that have passed.